spec/lib/healthchecks/queue_dead_set_check_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'sidekiq/testing'
+require 'okcomputer'
+require 'healthchecks'
+require 'scholarsphere/redis_config'
+
+RSpec.describe HealthChecks::QueueDeadSetCheck do
```

Comment:
do we need to add a `unless: !Scholarsphere::RedisConfig.new.valid?` ?

Reply:
good catch, this should match the other one.